src/utils/COMETAdapter.cpp

```diff
@@ -282,6 +282,14 @@ class TOPPCOMETAdapter :
       String term_distance = "-1";
       String nc_term = "0";
 
+      if (residues=="N-term")
```

Comment:
also note that this may change after https://github.com/OpenMS/OpenMS/pull/2504 has been merged